silk-core/src/main/kotlin/net/silkmc/silk/core/world/WorldExtensions.kt

```diff
@@ -0,0 +1,67 @@
+@file:Suppress("unused")
+
+package net.silkmc.silk.core.world
+
+import net.minecraft.world.entity.Entity
+import net.minecraft.world.entity.EntitySelector
+import net.minecraft.world.level.CollisionGetter
+import net.minecraft.world.level.EntityGetter
+import net.minecraft.world.level.Level
+import net.minecraft.world.level.entity.EntityTypeTest
+import net.minecraft.world.phys.AABB
+import net.minecraft.world.phys.shapes.VoxelShape
+import net.silkmc.silk.core.kotlin.asKotlinRandom
+import kotlin.random.Random
+
+val Level.kotlinRandom: Random
+    get() = random.asKotlinRandom()
+
+fun CollisionGetter.entityCollisions(
+    entity: Entity? = null,
+    box: AABB,
+): List<VoxelShape> {
+    return getEntityCollisions(entity, box)
+}
+
+fun CollisionGetter.collisions(
+    entity: Entity? = null,
+    box: AABB,
+): Iterable<VoxelShape> {
+    return getCollisions(entity, box)
+}
+
+fun CollisionGetter.blockCollisions(
+    entity: Entity? = null,
+    box: AABB,
+): Iterable<VoxelShape> {
+    return getBlockCollisions(entity, box)
+}
+
+fun EntityGetter.entityCollisions(
+    entity: Entity? = null,
+    box: AABB,
+): List<VoxelShape> {
+    return getEntityCollisions(entity, box)
+}
+
+fun EntityGetter.entities(
+    except: Entity? = null,
+    box: AABB,
+    predicate: (Entity) -> Boolean = EntitySelector.NO_SPECTATORS::test,
+): List<Entity> {
+    return getEntities(except, box, predicate)
+}
```

Comment:
don't these functions just remove "get" from the name of the original function?

Reply:
yeah basically, though it does also make them have an optional parameter for the `Entity`, which for all those methods if none is provided then it's ignored (I forget if there is an overload for this or not by default. I'd have to check.)

also, imo it feels a lot nicer without the `get`, but that's just me